cscs-checks/apps/lammps/lammps_check.py

```diff
@@ -14,14 +14,19 @@ def __init__(self, name, **kwargs):
         # Reset sources dir relative to the SCS apps prefix
         self.sourcesdir = os.path.join(self.current_system.resourcesdir,
                                        'LAMMPS')
-        self.sanity_patterns = sn.assert_found(r'Total wall time:',
-                                               self.stdout)
-
+        self.reference_energy = {
+            'daint:gpu': -4.6197356,
+            'daint:mc': -4.6195837,
+            'dom:gpu': -4.6189305,
+            'dom:mc': -4.619577
+        }
```

Comment:
Numerical sanity must not be related to a system. A result has a physical meaning or not regardless of the system.